fala/templates/adviser/scotland_warning.html

```diff
@@ -0,0 +1,11 @@
+<div class="govuk-grid-column-full">
```

Comment:
This should be `class="govuk-grid-column-two-thirds"` in order to get the text to wrap.

Reply:
Oooof! Good catch.